### ISSUE TITLE
[CPM] When using CPM, default to the version specified in Directory.Packages.props when adding a package in the NuGet UI

### ISSUE THREAD

**Issue by adcloeshelle** [NuGet/Home#12966]
### NuGet Product(s) Affected

Visual Studio Package Management UI

### Current Behavior

When you add a new NuGet package, the default version selected is the latest version.

### Desired Behavior

Ideally, the default version selected is the version specified in Directory.Packages.props.

### Additional Context

This will make it harder for developers to accidentally change the package version for all projects in the repository. It also makes it easy to know which version to select

**Comment by FrankLakmann** [comment]
Yes, this unexpected behavior tricked me, too. Please preselect the version from the Directory.Packages.props and give it a visual hint analogue to "latest stable:"  - something like "currently used in ths branch:". I know branch is not exactly correct but that is what it boils downto in practice.
In general, the dropdown list of the availabe versions needs much more info, e.g. hints for LTS-Versions.
This would help people to pick the sensible version of for example the Microsoft.Data.SqlClient without having to read the whole project docu (https://learn.microsoft.com/en-us/sql/connect/ado-net/sqlclient-driver-support-lifecycle?view=sql-server-ver16#long-term-support-lts-releases). Latest stable is not always the right choice.
